.github/workflows/main.yml

```diff
@@ -24,22 +41,25 @@ jobs:
         brew install popt
         brew install swig
         brew install upx
+        brew install libxmlsec1
+        brew install openssl
         npm install -g appdmg
-        echo "::add-path::/usr/local/opt/qt/bin"
 
     # Runs a set of commands using the runners shell
     - name: Run openSCAP build process
       run: |
-        git clone --depth 1 https://github.com/openscap/openscap.git -b master
+        export PATH="$PATH:/usr/local/opt/qt/bin"
+        git clone --depth 1 https://github.com/openscap/openscap.git -b maint-1.3
```

Comment:
Unless maint-1.3 has the macOS elements from the main branch, this shouldn't be maint-1.3.

Reply:
[`master` is 7 commits ahead, 473 commits behind `maint-1.3`](https://github.com/OpenSCAP/openscap/compare/master). The only material commit not in `maint-1.3` that is in `master` is:

https://github.com/OpenSCAP/openscap/commit/5a09d769deeeabb3b52f28013856ce049c04b2e3

But it looks like a similar change happened: https://github.com/OpenSCAP/openscap/blob/maint-1.3/CMakeLists.txt#L573

I think in both projects, development has moved off of `master` branch in favor of versioned branches.